Sources/Widgets/Widgets3D/SeedWidget/behavior.js

```diff
@@ -5,6 +5,7 @@ export default function widgetBehavior(publicAPI, model) {
 
   const moveHandle = model.widgetState.getMoveHandle();
   moveHandle.setVisible(true);
+  moveHandle.setDirection([0, 0, 1]);
```

Comment:
A nit: I'd recommend moving this to the `initialValues` object as a `direction` key in `state.js`.

Reply:
Thanks. Changed in [ae9d40e](https://github.com/Kitware/vtk-js/pull/2907/commits/ae9d40e5b88c4fd4ef8ce9db94b624d08d934456)